Update Hyperion documentation to include new effect sensor (#29429)
home-assistant/home-assistant.github.io: source/_integrations/hyperion.markdown

streamable sources will actually stream content (e.g., USB Capture Devices will work, but
static colors will not).

## Sensors

> A sensor (Visible Priority) provides the effect currently displayed by the Hyperion server for the selected instance. Attributes of this sensor allows to obtain more details on the nature of the effect as described in the [Hyperion API](https://docs.hyperion-project.org/en/json/ServerInfo.html#priorities)

Reviewer: ```suggestion
A sensor (Visible Priority) provides the effect currently displayed by the Hyperion server for the selected instance. Attributes of this sensor provide more details on the nature of the effect. For a detailed description, refer to the [Hyperion API](https://docs.hyperion-project.org/en/json/ServerInfo.html#priorities).
```
